var/spack/repos/builtin/packages/gtkplus/package.py

```diff
@@ -36,7 +36,7 @@ class Gtkplus(MesonPackage):
         deprecated=True,
     )
 
-    variant("cups", default=False, description="enable cups support")
+    variant("cups", default=True, description="enable cups support")
```

Comment:
Why this change? If another package requires `gtkplus+cups`, it should say that.

Reply:
Let's separate that out into a separate PR. Changing the default doesn't fix the issue, it just hides it.

Reply:
Done.  It's here: https://github.com/spack/spack/pull/42301